terms.html

```diff
@@ -5,9 +5,11 @@
 <dl class="termlist">
   <dt><dfn data-lt="claims">claim</dfn></dt>
   <dd>
-A digital representation of an assertion made about an <a>entity</a> by a specific <a>entity</a>.
-The entity about which the assertion is made is called the <a>subject</a> of the claim.
-The meaning of the claim (its semantics) is determined by its author.
+A digital representation of an assertion made, by a specific <a>entity</a>, about someone or something that exists.
+This 'someone or something that exists' is called the <a>subject</a> of the <a>claim</a>.
+This digital representation typically consists of an identifier (typically: an URI) 
```

Comment:
```suggestion
This digital representation typically consists of an identifier (usually, a URI) 
```

Reply:
This section is problematic, as subjects need not be "something that exists".  Subjects are not necessary.

Reply:
"something that exists" may be a "concept that exists", as all concepts do.

That "something that exists" may be identified by a URI or (explicitly or implicitly) by a blank node.

I don't understand the problem, if it's not addressed by the above.